types/storybook__addon-knobs/index.d.ts

```diff
@@ -0,0 +1,62 @@
+// Type definitions for @storybook/addon-knobs 3.0
+// Project: https://github.com/storybooks/storybook
+// Definitions by: Joscha Feth <https://github.com/joscha>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+/// <reference types="storybook__react" />
+
+// TODO: Once TS2.3 is released,
+// https://github.com/Microsoft/TypeScript/issues/14819 should be fixed.
+// Then, upgrade this package's typescript version to 2.3 and
```

Comment:
I just tried removing the wrappers, but now get:
```
/Users/joscha/Development/DefinitelyTyped/types/storybook__addon-knobs/index.d.ts
ERROR: 10:32  expect  TypeScript compile error: Cannot find module '@storybook/react'.

/Users/joscha/Development/DefinitelyTyped/types/storybook__addon-knobs/storybook__addon-knobs-tests.tsx
ERROR: 2:27   expect  TypeScript compile error: Cannot find module '@storybook/react'.
ERROR: 14:8   expect  TypeScript compile error: Cannot find module '@storybook/addon-knobs'.
```

I have used `/// <reference types="storybook__react" />` at the top of the file, but for some reason it still is not picked up correctly. Any idea what I might be doing wrong?